### PR TITLE
fix(editor): Use projects instead of users in resource filter

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/components/layouts/ResourcesListLayout.vue
Comment (krynble) [inline packages/editor-ui/src/components/layouts/ResourcesListLayout.vue:324]: Hey Csaba, please add the changes events to https://www.notion.so/n8n/List-of-telemetry-events-that-will-need-to-change-0359839a92444315b7c4030e07481ac4

File: packages/editor-ui/src/plugins/i18n/locales/en.json
Comment (krynble) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:2308]: We should discuss with JC because in the end, there are projects but personal projects will be displayed as people, so we might want to name it "Source" or "Origin" rather than "Projects". Since it's just a UI thing, it shouldn't block merging.

**Review by krynble — APPROVED** [review]

File: packages/editor-ui/src/components/layouts/ResourcesListLayout.vue
Comment (cstuncsik) [reply packages/editor-ui/src/components/layouts/ResourcesListLayout.vue:324]: Actually, the argument the function gets is not being sent; it was just to check if it wasn't a filter reset event.
When resetting, there is no telemetry call.

Other than that it just sends the whole filter state which is still happening but with some changes (`ownedBy` and `sharedWith` are removed from the object and `homeProject` added)

But I added it to the notion list just in case

File: packages/editor-ui/src/plugins/i18n/locales/en.json
Comment (cstuncsik) [reply packages/editor-ui/src/plugins/i18n/locales/en.json:2308]: Ok, I like **Origin** but will wait for the final decision

**Review by krynble — APPROVED** [review]

**Pull request merged**
